src/classmentors/services.js

```diff
@@ -605,7 +605,7 @@ export function clmDataStoreFactory(
         return ref.set(false);
       }
 
-    },
+        },
```

Comment:
You can use `./node_modules/.bin/eslint src/classmentors/services.js --fix` to reformat this file.